taxi/validators.py

```diff
@@ -0,0 +1,14 @@
+from django.core.exceptions import ValidationError
+
+
+class DriverValidationLicenseNumber:
```

Comment:
The `DriverValidationLicenseNumber` class should not be used as a mixin for forms. Instead, the `clean_license_number` method should be called within the form's `clean` method to perform validation.